app/models/zone.rb

```diff
@@ -194,6 +194,11 @@ def ems_clouds
     ext_management_systems.select { |e| e.kind_of?(EmsCloud) }
   end
 
+  # @return [Array<ExtManagementSystem>] All emses that can collect Capacity and Utilization metrics
+  def ems_collectable
```

Comment:
collectable here is very ambiguous.  We also "collect" inventory and metrics.

Reply:
metrics_collectable it is